evm_loader/program/src/account_storage/apply.rs

```diff
@@ -270,6 +282,72 @@ impl<'a> ProgramAccountStorage<'a> {
         Ok(())
     }
 
+    fn apply_withdrawals(
+        &mut self,
+        withdrawals: Vec<Withdraw>,
+        operator: &Operator<'a>,
+    ) -> Result<(), ProgramError> {
+        debug_print!("apply_withdrawals {:?}", withdrawals);
+
+        let (authority, bump_seed) = Pubkey::find_program_address(&[b"Deposit"], self.program_id);
+
+        let pool_address = get_associated_token_address(
+            &authority,
+            &crate::config::token_mint::id()
+        );
+
+        let signers_seeds: &[&[&[u8]]] = &[&[b"Deposit", &[bump_seed]]];
+
+        for withdraw in withdrawals {
+            let dest_neon = self.solana_accounts[&withdraw.dest_neon];
+
+            if dest_neon.data_is_empty() {
+                let create_acc_insrt = create_associated_token_account(operator.key,
+                                                                       &withdraw.dest,
+                                                                       &crate::config::token_mint::id());
+
+                let account_infos: &[AccountInfo] = &[
+                    (**operator).clone(),
+                    dest_neon.clone(),
+                    self.solana_accounts[&withdraw.dest].clone(),
+                    self.solana_accounts[&crate::config::token_mint::id()].clone(),
+                    self.solana_accounts[&spl_token::id()].clone(),
+                    self.solana_accounts[&rent::id()].clone(),
+                    self.solana_accounts[&spl_associated_token_account::id()].clone(),
+                ];
+
+                invoke_signed(&create_acc_insrt, account_infos, signers_seeds)?;

```

Comment:
`signer_seeds` is not needed for `create_associated_token_account`.
Also `invoke` can be used instead of `invoke_signed`